repo/fsrepo/migrations/migrations.go

```diff
@@ -16,6 +16,12 @@ import (
 
 var DistPath = "https://ipfs.io/ipns/dist.ipfs.io"
 
+func init() {
+	if dist := os.Getenv("IPFS_DIST_PATH"); dist != "" {
```

Comment:
i think this should be a variable, not an env var. env vars are really pernicious as is. but added here that this control the distribution updates, rough!

the issue with env vars and _why_ they're really bad for security is that it can turn secure scripts and commands into something unexpected by running before. meaning, in something like this:

```
> ./something
...
> /secure/bin/ipfs daemon --yes-run-migrations
```
`something` may not have been secured to not download anything, doesnt have permissions to change binaries or anything, but it can change an env var. it adds attach surface area.